spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -4009,8 +4041,21 @@ def in_comment(self, cursor=None):
         else:
             return False
 
-    def in_string(self, cursor=None):
+    def in_string(self, cursor=None, position=None):
+        """Returns true if the given position is inside a string
+
+        Overrides TextEditBaseWidget.in_string
+
+        Argument priority:
+            1. cursor
+            2. position
+            3. self.textCursor()

```

Comment:
Please make the same changes here as the ones I suggested for the docstring of `in_comment` method above.